src/PostgREST/Auth.hs

```diff
@@ -20,33 +20,40 @@ module PostgREST.Auth
 import qualified Crypto.JWT          as JWT
 import qualified Data.Aeson          as JSON
 import qualified Data.HashMap.Strict as M
+import qualified Data.Text           as T
 import qualified Data.Vector         as V
 
-import Control.Lens            (set)
-import Control.Monad.Except    (liftEither)
-import Data.Either.Combinators (mapLeft)
-import Data.Time.Clock         (UTCTime)
+import Control.Lens              (set)
+import Control.Monad.Except      (liftEither)
+import Data.Either.Combinators   (mapLeft)
+import Data.List                 (lookup)
+import Data.Time.Clock           (UTCTime)
+import Network.HTTP.Types.Header (hAuthorization)
+import Network.Wai               (Request (..))
```

Comment:
Uh. This needs to be imported qualified. I am terribly confused by `Request` below, which is **not** our `Request`. `Wai.Request` would make that clear.

Reply:
Fixed!